app/repository/pg/user.go

```diff
@@ -32,20 +32,38 @@ func NewRepo(db *sql.DB) *Repo {
 // Add meth implements usecase.UserRepository interface
```

Comment:
Bad godoc. I'm surprised linters don't highlight this. Did you check your linters? Are they alive?

Reply:
Yes, they are alive. They skiped "meth". Fixed myself

Reply:
The godoc is still bad. Must start with `AddUser`

Reply:
fixed